### ISSUE TITLE
Date not saving

### ISSUE THREAD

**Issue by FilipeOS** [carmoreira/date-time-picker-field#7]
Hi,

The date/time is not saving on database, it's showing `00:00 30/11/-1`

My input `<input name="booking_service_date" id="booking_service_date" type="date" class="datepicker" autocomplete="off" required/>`

I tried to save directly using $_POST but did not work still so I tried to convert to date before:
```
$newDate = date('Y-m-d H:i:s', strtotime($_POST['booking_service_date']));
'booking_service_date' => $newDate,
```

On my mysql database I have the table like `booking_service_date | datetime` (not string).

Basically it saves but never like this... I tried to save on datetime, maybe I change to varchar and forget?

**Comment by carmoreira** [comment]
Hi
You should use type="text" to disable any validation your browser might impose. 
You should also save it as a string, unless the format you're using is mysql compatible or you convert it to one after the post. 
Greetings, Carlos
